sql/redshift/001_initial.schema.up.sql

```diff
@@ -0,0 +1,40 @@
+BEGIN;
+CREATE TABLE rpc_endpoint_requests(
+    id varchar(128) not null distkey,
+    received_at timestamptz not null default sysdate,
```

Comment:
not sure why, but my comment here disappeared: https://github.com/flashbots/rpc-endpoint/pull/62/files#diff-49621259d00261dd2798c920f922ceb9411562f3ef333791864b480f61a32446R4 -- received_at should not use default sysdate because it should be set by the application

Reply:
updated